submissions/jfrank/.nvmrc

```diff
@@ -0,0 +1 @@
+4.0.0
```

Comment:
The long term support version of Node is at 4.2.4 -- thoughts on bumping this to 4.2.4?